packages/components/src/components/DetailsHeader/DetailsHeader.js

```diff
@@ -16,11 +16,45 @@ import { ChevronRight24 as DefaultIcon } from '@carbon/icons-react';
 import { injectIntl } from 'react-intl';
 import { getStatus } from '@tektoncd/dashboard-utils';
 
-import { StatusIcon } from '..';
+import { FormattedDuration, StatusIcon } from '..';
 
 import './DetailsHeader.scss';
 
 class DetailsHeader extends Component {
+  getDuration() {
```

Comment:
Followed up in DM but might be worth making this configurable ?

Reply:
good point, I also noticed some odd behaviour with the duration being revised downwards when a step completes so I'll update this to only display on completion